sites/fast-creator/app/configs/fast/library.fast.registry.ts

```diff
@@ -0,0 +1,13 @@
+import { DesignSystem } from "@microsoft/fast-foundation";
+import { allComponents } from "@microsoft/fast-components";
+import { setupFASTComponentDesignSystem } from "./library.fast.design-system.mapping";
+
+export function registerFASTComponents(): void {
+    setupFASTComponentDesignSystem(document.body);
+
+    DesignSystem.getOrCreate().register(
```

Comment:
Before calling `register` you can call `withPrefix` to set the prefix for all registered components, rather than setting it on each individual component.